guides/v2.3/graphql/reference/products.md

```diff
@@ -85,12 +86,21 @@ updated_at
 weight
 ```
 
-<div class="bs-callout bs-callout-info" id="info" markdown="1">
 The following attributes are not used in responses:
+
 * `or` - The keyword required to perform a logical OR comparison.
 * `news_from_date` - This attribute is transformed to `news_from_date` in a response.
 * `news_to_date` - This attribute is transformed to `news_to_date` in a response.
-*
+
+<div class="bs-callout bs-callout-info" id="info" markdown="1">
+GraphQL automatically filters out a product attribute if ALL of the following fields are set to **No** on the attribute's Storefront Properties page in Admin:
+
+* Comparable on Storefront
+* Use in Layered Navigation
+* Use in Search Results Layered Navigation
+* Visible on Catalog Pages on Storefront
+* Used in Product Listing
+* Used for Sorting in Product Listing
 </div>
```

Comment:
Use the liquid syntax instead of the HTML:

{% 
include note.html
type='info'
content='GraphQL automatically filters out a product attribute if ALL of the following fields are set to **No** on the attribute's Storefront Properties page in Admin:
* Comparable on Storefront
* Use in Layered Navigation
* Use in Search Results Layered Navigation
* Visible on Catalog Pages on Storefront
* Used in Product Listing
* Used for Sorting in Product Listing'
%}

Otherwise, looks good.